crates/p2p_protocol/protocol/p2p.capnp

```diff
@@ -0,0 +1,179 @@
+# lib3h p2p wire protocol
+# note, initial handshake will use transit-enc.capnp
+# once that is complete, messages from this schema
+# will be encoded as negotiated.
+
+@0x859198991b95d3e1;
+
+struct Message {
```

Comment:
Suggest renaming to `P2pMessage` as `Message` sounds too generic and also collisions with `Message` in `transport_dm`.

Reply:
:+1: